src/applications/claims-status/containers/ClaimStatusPage.jsx

```diff
@@ -128,7 +126,19 @@ const generateEventTimeline = claim => {
   const supportingDocuments = generateSupportingDocuments(claim);
   const trackedItems = generateTrackedItems(claim);
 
-  const events = [...trackedItems, ...supportingDocuments, ...phases];
+  // The 'filed' event is a bit of a special case,
+  // We want it to the the last item so that it
+  // gets associated with phase 1 (See the last if
+  // statement in this function for more detail)
+  const events = [
+    ...trackedItems,
+    ...supportingDocuments,
+    ...phases,
+    {
```

Comment:
There was an edge case where a claim whose `claimDate` was the same as it's `phase1CompleteDate` would end up not showing an event in the timeline, this should fix that. It's important to note that the events are sorted in reverse chronological order